src/SignalR/server/Specification.Tests/src/HubLifetimeManagerTestBase.cs

```diff
@@ -8,12 +8,23 @@
 
 namespace Microsoft.AspNetCore.SignalR.Specification.Tests
 {
+    /// <summary>
+    /// Base test class for lifetime manager implementations. Nothing specific to scale-out for these tests.
+    /// </summary>
+    /// <typeparam name="THub">The type of the <see cref="Hub"/>.</typeparam>
     public abstract class HubLifetimeManagerTestsBase<THub> where THub : Hub
     {
-        public HubLifetimeManager<THub> Manager { get; set; }
```

Comment:
Technically a breaking change, but no one uses this package so... 🤷

Reply:
Let's keep this.